pkg/sensors/exec/userinfo/userinfo.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
```

Comment:
The first commit is named  "tetragon: resolve uid->username for processes from /proc", but it seems that it just refactors the code for subsequent patches. Please rename the commit. Also, all of the patches have a single tile. Please add some context what each patch does and why.

Reply:
Actually it is used in this patch, but improved git log.